SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -23,9 +40,13 @@ void SpatialEventTracer::TraceCallback(void* UserData, const Trace_Item* Item)
 	}
 
 	uint32_t ItemSize = Trace_GetSerializedItemSize(Item);
-	if (EventTracer->BytesWrittenToStream + ItemSize <= EventTracer->MaxFileSize)
+	if (EventTracer->MaxFileSize == 0 || (EventTracer->BytesWrittenToStream + ItemSize <= EventTracer->MaxFileSize))
```

Comment:
Rather than manual file size tracking here for single-log, could we use rotating and set the max file count to 1 @improbable-jonphillips?

Reply:
@improbable-danny sorry I missed this comment. I did consider this (when discussing this on the runtime side) but there were a couple of reasons I didn't think we should do this:
* (reason not to use the rotating file stream) the rotating file stream will put a timestamp in the filename and we simply don't need it
* (reason to expose different user-facing options for rotating event logs vs. single event log) makes the 2 different but equally valid use cases/workflows more obviously distinct in terms of the UX
I think you were probably just asking about the first point, though.